pipeline/src/main/java/edu/illinois/cs/cogcomp/pipeline/server/MainServer.java

```diff
@@ -175,7 +173,7 @@ public static void main(String[] args) {
     }
 
     private static String annotateText(AnnotatorService finalPipeline, String text, String views,
-            Logger logger) throws AnnotatorException {
+                                       Logger logger) throws AnnotatorException {
```

Comment:
Just curious: why is Logger a method argument, not a static class member? (the latter is more standard in CCG code)

Reply:
Don't remember it now, but it seems odd. Fixed it.